pythonscript/embedded/godot/basis.py

```diff
@@ -48,6 +48,9 @@ def build_from_axis_and_angle(cls, axis, phi):
 
     def __init__(self):  # TODO: allow rows as param ?
         self._gd_ptr = godot_basis_alloc()
+        self.x = Vector3(1, 0, 0)
```

Comment:
It would be more optimal to directly use `godot_basis_new_with_rows` (https://github.com/godotengine/godot/blob/master/modules/gdnative/gdnative/basis.cpp#L40), this way we avoid all the wrapper overhead.

BTW In the future when we will look for performance it could worth to try to tweak the GDnative API to allow constructors taking 9 floats to avoid initializing temporary data.